### DISCUSSION TITLE
Which southern Italian city is credited as the birthplace of the modern pizza?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#516] [Galaxy Trivia] [answered]
**Bonus:** Describe the pizza dough recipe regulated by AVPN?

**Comment by peterbaccaro** [comment]
**Napoli (aka Naples)** is the birthplace of pizza.  I have relatives in Naples and the best pizzas are from there as the best mozzarellas are made in that area of Italy and tomatoes are grown locally so ingredients are very fresh and pizzas are cooked in wood burning ovens.

This will make 6 Neapolitan pizzas of 11-12 inches:
922g Tipo 00 flour (100%)
553ml cool water (60%)
23g of fine sea salt (2.5%)
1g of fresh yeast (0.1%) or 0.28g of active dry yeast (0.03%)

**Comment by thuongtruong1009** [comment, marked as answer]
The southern Italian city credited as the birthplace of the modern pizza is Naples. The traditional Neapolitan pizza, also known as pizza Napoletana, is made with San Marzano tomatoes, mozzarella cheese made from the milk of water buffalo raised in the Campania region, and fresh basil, on a thin crust that is crispy on the outside and soft and chewy on the inside. The Margherita pizza, which is topped with tomato sauce, mozzarella cheese, and basil, was created in Naples in 1889 by pizzaiolo Raffaele Esposito in honor of Queen Margherita of Savoy. Pizza Napoletana has been recognized as a UNESCO Intangible Cultural Heritage since 2017.
